packages/liveblocks-node/rollup.config.js

```diff
@@ -0,0 +1,217 @@
+import babelPlugin from "@rollup/plugin-babel";
+import commandPlugin from "rollup-plugin-command";
+import dts from "rollup-plugin-dts";
+import path from "path";
+import prettierPlugin from "rollup-plugin-prettier";
+import replaceText from "@rollup/plugin-replace";
+import resolve from "@rollup/plugin-node-resolve";
+import { terser as terserPlugin } from "rollup-plugin-terser";
+import typescriptPlugin from "@rollup/plugin-typescript";
+import { promises } from "fs";
+const babelConfig = require("./babel.config");
+
+function execute(cmd, wait = true) {
+  return commandPlugin(cmd, { exitOnFail: true, wait });
+}
+
+function stripComments() {
+  return terserPlugin({ mangle: false, output: { comments: false } });
+}
+
+function prettier() {
+  return prettierPlugin({ parser: "typescript", tabWidth: 2 });
+}
+
+/**
+ * TypeScript plugin configured to only produce *.js code files, no *.d.ts
+ * files.
+ */
+function typescriptCompile() {
+  return typescriptPlugin({
+    declaration: false,
+    tsconfig: "./tsconfig.build.json",
+    noEmitOnError: true, // Let rollup build fail if there are TypeScript errors
+  });
+}
+
+/**
+ * TypeScript plugin configured to only produce *.d.ts files, no code
+ * compilation.
+ */
+function typescriptDeclarations(outDir) {
+  return typescriptPlugin({
+    declaration: true,
+    outDir,
+    tsconfig: "./tsconfig.build.json",
+    noEmitOnError: true, // Let rollup build fail if there are TypeScript errors
+  });
+}
+
+function getBabelOptions(extensions, targets) {
+  return {
+    ...babelConfig,
```

Comment:
a simple object spread unlike for liveblocks-client, where the config is a function